### DISCUSSION TITLE
How Does Copilot Know Song Lyrics?

### DISCUSSION THREAD

**Discussion by Petercopter** [community/community#54176] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

This is an actual screenshot sent to me by a friend.
![image](https://user-images.githubusercontent.com/1493974/235498290-7e35de7e-628b-4584-b152-290a9a48b153.png)
1. LOL
2. How does copilot know this?

**Comment by orihpt** [comment]
I would also like to know! At first I thought it knows the lyrics of Never Gonna Give You Up because of the memes, but later I found out that it also knows lyrics of a lot more songs.
